microsoft-edge/extensions-chromium/publish/aad-account.md

```diff
@@ -1,75 +1,145 @@
+
+
+<!-- ====================================================================== -->
+## Making sure you have a Partner Center account with a Microsoft account (MSA) as the Primary Owner
+
+To create a Partner Center account that can publish Microsoft Edge extensions, you must have a Microsoft account (MSA), either by creating one directly or by using your personal GitHub account credentials.  
+
+After your Partner Center account is able to publish Microsoft Edge extensions, you can link the Partner Center account to an Azure Active Directory (Azure AD) tenant.  An linked Active Directory tenant enables the added users to login to your Partner Center developer account by using their work accounts.
+
+Different programs on Partner Center require different types of accounts:
+
+*  The Microsoft Edge program (like the Windows developer program) requires a Partner Center _developer_ account.  A Partner Center developer account has a Primary Owner that's a Microsoft account (MSA).
+
+*  In contrast, Azure marketplace requires a Partner Center _commercial_ account.  (To enroll, the user has to login with their work account.)
+
+Also see these articles in the Partner Center documentation:
+*  [Manage your Partner Center account](/partner-center/partner-center-account-setup)
+*  [Microsoft Partner Network membership benefits](/partner-center/mpn-overview)
+
+
+<!-- ====================================================================== -->
 ## Step 1: Enroll in the Microsoft Edge program on Partner Center
 
-If you do not have an account and are new to Partner Center, continue to Step **1**. If you have a Partner Center developer account, continue to Step **2**.
+First, determine whether you have a Partner Center account.  If you have a Partner Center account, determine whether the Primary Owner is a Microsoft Account (MSA), which is required for joining the Microsoft Edge program, to manage Microsoft Edge extensions.  Then follow the steps in the section that applies to your type of Partner Center account.
+
+### If you don't have a Partner Center account
+
+1.  Use a Microsoft account (MSA) to register with the Microsoft Edge program, by following the steps in the article [Register as a Microsoft Edge extension developer][DeveloperRegistration].<!-- = create-dev-account.md-->  As mentioned in that article, you can use your GitHub account to create a Microsoft account (MSA).
+
+Next, do [Step 2: Associate Azure Active Directory with your Microsoft Edge Program account](#step-2-associate-azure-active-directory-with-your-microsoft-edge-program-account) below.
+
+
+### If the Primary Owner of your Partner Center account isn't a Microsoft account (MSA)
+
+For a Partner Center account to manage Microsoft Edge extensions, the Primary Owner of the Partner Center account must be a Microsoft account (MSA).
+
+If the Primary Owner of a Partner Center account isn't a Microsoft account (MSA):
```

Comment:
This sentence seems incorrect. Rather we should say if they already know that the owner is not an MSA, the next steps are invalid. Seems like this statement should be:  - To find the owner of the Partner Center account that you are logged in.